src/components/copy-to-clipboard.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable max-len */
+import React from "react";
+import ReactDOM from "react-dom";
+import Clipboard from "clipboard";
+
+export default class CopyToClipboard extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    this.clipboard = new Clipboard(ReactDOM.findDOMNode(this.refs.copyToClipboard));
+  }
+
+  componentWillUnmount() {
+    this.clipboard.destroy();
+  }
+
+  render() {
+    return (
+      <button
+        ref="copyToClipboard"
```

Comment:
I don't believe the ref-callback style will work for this implementation. As the component re-renders, the function is re-invoked each time. And because we are creating a `new Clipboard()`, the re-render breaks the implementation.

This is a great style to know for future implementations, but isn't right for this scenario from what I can tell.